pypolyclip/pypolyclip.py

```diff
@@ -127,7 +127,7 @@ def single(x, y, nxy):
         The y coordinates of the polygon corners
 
     nxy : list, tuple, or `np.ndarray`
-           The size of the pixel grid.
+        The size of the pixel grid.
 
```

Comment:
```suggestion

return_polygons : bool
    If `True` then return the x and y coordinates of the vertices of the 
    newly-clipped polygons.
```

Reply:
The "Returns" section also needs descriptions of the returned `px` and `py` variables.